config_src/drivers/nuopc_cap/mom_cap.F90

```diff
@@ -1807,7 +1816,18 @@ subroutine ModelAdvance(gcomp, rc)
       if (ChkErr(rc,__LINE__,u_FILE_u)) return
     end if
 
-    if (write_restart .or. write_restartfh) then
+    write_restart_eor = .false.
+    if (restart_eor) then
+      if (ESMF_AlarmIsRinging(stop_alarm, rc=rc)) then
+         if (ChkErr(rc,__LINE__,u_FILE_u)) return
+         write_restart_eor = .true.
+         ! turn off the alarm
+         call ESMF_AlarmRingerOff(restart_alarm, rc=rc )
```

Comment:
This needs to be turning off the stop_alarm, not the restart_alarm.

Reply:
@DeniseWorthen Ok, code has been changed and tested.